### PR TITLE
Tenant bootstrapping

### PR DESCRIPTION
**Pull request by cvalaas**

### CODE REVIEW

File: google_tenant_project_bootstrap/main.tf [hunk 1]
```diff
@@ -0,0 +1,26 @@
+module "google_gke_tenant" {
+  source = "github.com/mozilla/terraform-modules//google_gke_tenant?ref=main"
+
+  project_id         = var.project
+  environment        = var.environment
+  cluster_project_id = var.gke_cluster_project_id
+  namespace          = var.application
```
Comment (jasonthomas) [inline google_tenant_project_bootstrap/main.tf:7]:
```suggestion
  application        = var.application
```
A namespace is a combination of $application-$env, so I think we just need to adjust this here. I did a bad job of reviewing https://github.com/mozilla/terraform-modules/blob/main/google_gke_tenant/variables.tf#L19-L24 so it should be adjusted there as well.

File: google_tenant_project_bootstrap/variables.tf [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+  description = "Name of infrastructure realm (e.g. prod or nonprod)."
+  type        = string
+
+  validation {
+    condition     = contains(["mgmt", "nonprod", "prod"], var.realm)
+    error_message = "Valid values for realm: mgmt, nonprod, prod."
+  }
+}
+
+variable "region" {
```
Comment (jasonthomas) [inline google_tenant_project_bootstrap/variables.tf:17]: Is region being used in any of the modules?

File: google_tenant_project_bootstrap/variables.tf [hunk 3]
```diff
@@ -0,0 +1,55 @@
+variable "project" {
+  default     = null
+  description = "The project ID in which we're doing this work."
+  type        = string
+}
+
+variable "realm" {
+  description = "Name of infrastructure realm (e.g. prod or nonprod)."
```
Comment (jasonthomas) [inline google_tenant_project_bootstrap/variables.tf:8]:
```suggestion
  description = "Name of infrastructure realm (e.g. prod, nonprod, mgmt, or global)."
```

File: google_tenant_project_bootstrap/variables.tf [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  type        = string
+
+  validation {
+    condition     = contains(["mgmt", "nonprod", "prod"], var.realm)
```
Comment (jasonthomas) [inline google_tenant_project_bootstrap/variables.tf:12]:
```suggestion
    condition     = contains(["mgmt", "nonprod", "prod", "global"], var.realm)
```

**Review by jasonthomas — DISMISSED** [review]
r+wc let me know if you have any questions.

**Comment by cvalaas** [comment]
@jasonthomas 's nit have been addressed

**Review by jasonthomas — APPROVED** [review]

**Pull request merged**